styles.css

```diff
@@ -161,6 +160,296 @@
   }
+
+  /* My Works */
+  .works {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 64px 24px;
+    gap: 58px;
+    position: relative;
+    width: 100%;
+    height: 3315px;
+    left: 0;
+    top: 0;
+  }
+
+  .works-heading {
+    width: 100%;
+    height: 97px;
+    font-family: Inter, 'Times New Roman', Times, serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    align-items: center;
+    text-align: center;
+    color: #172b4d;
+  }
+
+  .works-heading::after {
+    content: '';
+    display: block;
+    border-radius: 10px;
+    width: 50px;
+    height: 5px;
+    background: #36b37e;
+    align-items: center;
+    margin: 15px auto;
+  }
+
+  .works-container {
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    grid-template-columns: 100%;
+    grid-template-rows: 16.666% 16.666% 16.666% 16.666% 16.666% 16.666%;
+  }
+
+  .works-samples {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 474px;
+    max-height: fit-content;
+    list-style: none;
+    margin-bottom: 30px;
+    border-radius: 8px;
+    box-shadow: 0 4px 4px rgba(0, 0, 0, 0.5);
+    border: 1px solid #d0d9d4;
+  }
+
+  .works-samples .screenshot {
+    width: 100%;
+    height: 252px;
+    object-fit: cover;
+    border-radius: 8px;
+  }
+
+  .works-samples .screenshot .works-samples-img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    border-radius: 8px;
+  }
+
+  .works-samples .work-title {
+    padding: 16px;
+    width: 295px;
+    height: 57px;
+    font-family: Inter, 'Times New Roman', Times, serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 32px;
+    text-align: center;
+    margin: auto;
+    color: #3a4a42;
+  }
+
+  .works-samples .languages-container {
+    list-style-type: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    width: 100%;
+    height: 32px;
+    margin-top: 16px;
+  }
+
+  .works-samples .languages-container .languages {
+    gap: 8px;
+    background-color: #ebf0ee;
+    padding: 8px 12px;
+    height: 32px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border-radius: 4px;
+    font-family: Inter, 'Times New Roman', Times, serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+  }
+
+  .works-samples .see-project {
+    display: flex;
+    align-items: center;
+    text-align: center;
+    margin: 16px auto;
+    background-color: #36b37f;
+    color: white;
+    border-radius: 10px;
+    padding: 12px;
+    bottom: 8px;
+    font-family: Inter, 'Times New Roman', Times, serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+  }
+
+  /* About Me */
+
+  .aboutme {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```

Comment:
- [ ] Kindly add more padding on the top and the bottom of the paragraph here to match the figma design: 
yours:
![image](https://user-images.githubusercontent.com/91486791/235227834-4ca4f9a2-c8d3-4100-86d7-210455243d59.png)
figma: 
![image](https://user-images.githubusercontent.com/91486791/235227916-a0a90fa8-91fc-49b1-8232-f2a4a234b663.png)